tgui/packages/tgui/interfaces/GPS.tsx

```diff
@@ -0,0 +1,214 @@
+          <Stack>
+            <Stack.Item>
+              <LabeledList>
+                <LabeledList.Item label="Coordinates">
+                  {src_x}, {src_y}
+                </LabeledList.Item>
+                <LabeledList.Item label="Identifier">
+                  <Button
+                    onClick={() => act('change_identifier')}
+                    tooltip={gpsTooltip}
+                  >
+                    {src_name.slice(5, src_name.length)}
+                  </Button>
+                </LabeledList.Item>
+                <LabeledList.Item label="Coords">
+                  x:{' '}
+                  <NumberInput
+                    value={track_x}
+                    width={'3'}
+                    minValue={1}
+                    maxValue={300}
+                    step={1}
+                    onChange={(x_val: number) => act('set_x', { x: x_val })}
+                  />
+                  y:{' '}
+                  <NumberInput
+                    value={track_y}
+                    width={'3'}
+                    minValue={1}
+                    maxValue={300}
+                    step={1}
+                    onChange={(y_val: number) => act('set_y', { y: y_val })}
+                  />
+                  <Button
+                    mt={0.5}
+                    onClick={() =>
+                      act('track_coords', { x: track_x, y: track_y })
+                    }
+                  >
+                    Track
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </Stack.Item>
+            <Stack.Item>
+              <LabeledList>
+                <LabeledList.Item label="Trackable">
+                  <Button.Checkbox
+                    onClick={() => act('toggle_trackable')}
+                    selected={trackable}
+                    checked={trackable}
+                  >
+                    {trackable ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Send distress">
+                  <Button.Checkbox
+                    onClick={() => act('toggle_distress')}
+                    selected={distress}
+                    checked={distress}
+                  >
+                    {distress ? 'Yes' : 'No'}
+                  </Button.Checkbox>
+                </LabeledList.Item>
+                <LabeledList.Item label="Tracking">
+                  <Button onClick={() => act('track_gps')}>
+                    {tracking ? tracking : 'None'}
+                  </Button>
+                </LabeledList.Item>
+              </LabeledList>
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <Section title="Tracking">
+          <Collapsible title="GPS Devices">
+            {!!gps_info.length && (
+              <Stack vertical>
+                <TrackableList gps_info={gps_info} />
+              </Stack>
```

Comment:
Probably move the `Stack` into the `TrackableList` component's render, as it's an implementation detail of that, I think?